### ISSUE TITLE
BCM94360CD internal hub is not working

### ISSUE THREAD

**Issue by AIOFrame** [dhckdgjs/ASUS-PRIME-Z390-A-HACKINTOSH-iGPU-with-dGPU-UHD630-RX580#6]
First of all, amazing work and am really sorry for creating an issue.

I have this BCM94360CD, [AliExpress Link](https://www.aliexpress.com/item/4000027376561.html?spm=a2g0s.9042311.0.0.4cea4c4dPDrd7o) and I am unable to get bluetooth to work. Wifi works fine though, last time I tried to fix this, I got it working then lost all my ports and have been using my PC with just 2 ports.

**Comment by dhckdgjs** [comment]
A faulty USB port map causes USB malfunction. So it related to BT problems basically also.
Please use my port map in the most recent EFI and connect the connector (from PCI-e WiFi/BT card) to the HS05 port ;)

**Comment by AIOFrame** [comment]
Yes i resolved this just now by re-configuring USB ports. Thanks for your prompt response!

**Comment by AIOFrame** [comment]
Hey can you delete this issue ticket please ?

**Comment by ClassicOldSong** [comment]
Why bother deleting? Leave it here it may help others that have the same problem.

**Comment by AIOFrame** [comment]
No actually this is my company account, so would you please ?

**Comment by AIOFrame** [comment]
Hi @dhckdgjs can you please delete this issue? its showing up on official account of AIO. Please do the needful, thanks.
